### DISCUSSION TITLE
Badges are awesome

### DISCUSSION THREAD

**Discussion by idrayv** [community/community#73483] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Thanks GitHub, I love the badges idea, please implement more of them, gamification is something I love and what brings me additional motivation.

**Comment by JonSnow989** [comment]
That's really good and interesting tho

**Comment by anharsaja** [comment]
i wanna badges too

**Comment by idrayv** [comment, marked as answer]
More badges more happy programmers :)
